README.md

```diff
@@ -1,10 +1,29 @@
-[![Build Status](https://travis-ci.org/kuzzleio/kuzzle-plugin-cluster.svg?branch=master)](https://travis-ci.org/kuzzleio/kuzzle-plugin-cluster) [![codecov.io](http://codecov.io/github/kuzzleio/kuzzle-plugin-cluster/coverage.svg?branch=master)](http://codecov.io/github/kuzzleio/kuzzle-plugin-cluster?branch=master)
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/7868838/66303815-2617b000-e8fc-11e9-8c3f-613574be1746.png"/>
+</p>
+<p align="center">
+  <a href="https://david-dm.org/kuzzleio/kuzzle-plugin-cluster">
+    <img src="https://david-dm.org/kuzzleio/kuzzle-plugin-cluster.svg" />
+  </a>
+  <a href="https://travis-ci.com/kuzzleio/kuzzle-plugin-cluster">
+    <img alt="undefined" src="https://travis-ci.com/kuzzleio/kuzzle-plugin-cluster.svg?branch=master">
+  </a>
+  <a href="https://codecov.io/gh/kuzzleio/kuzzle-plugin-cluster">
+    <img src="https://codecov.io/gh/kuzzleio/kuzzle-plugin-cluster/branch/master/graph/badge.svg" />
+  </a>
+  <a href="https://github.com/kuzzleio/kuzzle-plugin-cluster/blob/master/LICENSE">
+    <img alt="undefined" src="https://img.shields.io/github/license/kuzzleio/kuzzle-plugin-cluster.svg?style=flat">
+  </a>
+</p>
 
-# Cluster mode plugin
 
-This plugin adds a cluster mode to Kuzzle.
+## About
 
-## Kuzzle
+### Kuzzle Cluster Plugin
+
+This plugin adds a masterless cluster mode to Kuzzle.
```

Comment:
You can add a link to the scalability guide here (like in the other PR)

Reply:
Done :slightly_smiling_face: